roles/kubernetes-apps/helm/tasks/main.yml

```diff
@@ -2,18 +2,34 @@
 - name: Helm | Make sure HELM_HOME directory exists
   file: path={{ helm_home_dir }} state=directory
 
-- name: Helm | Set up helm launcher
+- name: Helm | Remove helm
+  file:
+    path: "{{ bin_dir }}/helm"
+    state: absent
```

Comment:
bin_dir == : ${HELM_INSTALL_DIR:="/usr/local/bin"}

Both the script and kubespray will deploy helm to `/usr/local/bin` ( or /opt/bin in CoreOS ). By removing helm here it will cause this task to not be idempotent.